src/tdsops.f90

```diff
@@ -169,7 +178,7 @@ subroutine deriv_1st(self, delta, scheme, bc_start, bc_end, sym)
 
     self%dist_sa(:) = alpha; self%dist_sc(:) = alpha
 
-    n = self%n
+    n = self%mesh%get_n(self%dir, VERT)
```

Comment:
Can't really assume its `VERT` here, this will lead to wrong results. Depends on the type of the operator. `p2v`/`v2p` etc. I think its best resolving this before an instance of `tdsops_t` is created.